### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by salbertson**
This changes updates guidelines, where we need help and Copyright year.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,24 +81,27 @@ First, thank you for contributing!
...
+6. Celebrate!
 
 There a couple areas we would like to concentrate on.
 
-1. Add support for JavaScript
-2. Add support for CSS and Sass
-3. Write [style guides](app/models/style_guide) that don't currently exist and
-   would enforce the
-   [thoughtbot style guide](https://github.com/thoughtbot/guides).
+1. ~~Add support for JavaScript~~
```
Comment (gylaz) [inline README.md]: Should we remove this? Or are you thinking along the lines of [JSCS](http://jscs.info/) support?

**Comment by gylaz** [comment]
LGTM.

File: README.md (hunk 1 above)
Comment (salbertson) [inline README.md]: I added a strikethrough, just to show progress. What do you think?

Comment (thorncp) [inline README.md]: How about using a task list?

https://github.com/blog/1825-task-lists-in-all-markdown-documents

**Pull request merged**